src/igvfd/schemas/open_reading_frame.json

```diff
@@ -132,6 +132,9 @@
         }
     },
     "fuzzy_searchable_fields": [
+        "aliases",
+        "lab.title",
+        "submitted_by.title",
```

Comment:
I noticed you were alphabetizing in the other schemas, I think it would make sense to move "orf_id" to alphabetical order too

Reply:
Moved. Also I removed orf_id from exact search since its redundant there.